### PR TITLE
Horizon Angular bump - build overrides

### PR DESCRIPTION
**Pull request by darmach**
Added build overrides to kolla.yml required to fix CVE-2019-10768 Related to: https://bugs.launchpad.net/horizon/+bug/1997545

### CODE REVIEW

**Comment by darmach** [comment]
WIP: still need to build and test this

File: etc/kayobe/kolla.yml [hunk 1]
```diff
@@ -132,6 +132,10 @@ kolla_sources:
     type: git
     location: https://github.com/stackhpc/cloudkitty-dashboard.git
     reference: stackhpc/{{ openstack_release }}
+  horizon:
+    type: git
+    location: https://github.com/stackhpc/horizon.git
+    reference: bump-angular-yoga
```
Comment (mnasiadka) [inline etc/kayobe/kolla.yml]: After it's tested, let's merge the PR in Horizon and update this to stackhpc/yoga.

File: etc/kayobe/kolla.yml [hunk 2]
```diff
@@ -258,6 +262,11 @@ kolla_build_blocks:
     RUN sed -i 's/"pc-q35-rhel8.5.0"/"pc-q35-*"/' /usr/share/qemu/firmware/50-edk2-ovmf-cc.json
     {% endif %}
     {% endraw %}
+  horizon_header: |
+    # Fix for https://bugs.launchpad.net/horizon/+bug/1997545
```
Comment (mnasiadka) [inline etc/kayobe/kolla.yml:364]: worth a note here that the version comes from Zed u-c

**Comment by markgoddard** [comment]
Is it ready?

**Comment by darmach** [comment]
@markgoddard It's been built, and being tested on EBI DEV. Still need to test ironic on smslab, didn't want t cause disturbances during this week workshop

**Comment by darmach** [comment]
@markgoddard Finished testing, looks fine.

File: etc/kayobe/kolla.yml (hunk 2 above)
Comment (darmach) [reply etc/kayobe/kolla.yml:364]: Done

File: etc/kayobe/kolla.yml (hunk 1 above)
Comment (darmach) [reply etc/kayobe/kolla.yml]: Done

**Comment by mnasiadka** [comment]
Can you build the images and put tags in kolla/globals.yml?

**Comment by darmach** [comment]
Reworked stackhpc/horizon/yoga:
 - rebased Angular JS 1.8.2 bump related commits (wrong order, breaking upstream backports compatibility)
 - re-picked needed commits
 - synced with upstream
 
Need to rebase this and build containers

**Comment by mnasiadka** [comment]
Rocky 8 is failing on MariaDB

**Comment by markgoddard** [comment]
Seems like this one got stuck, and the images are probably now stale. How should we handle it?

**Comment by Alex-Welsh** [comment]
Not sure If we still need this but I've refreshed this branch [here](https://github.com/stackhpc/stackhpc-kayobe-config/tree/yoga-horizon-angular-cve-2019-10768-refresh) if it's still a problem we can merge, otherwise we can drop the PR

**Pull request closed without merging**